docs/README.md

```diff
@@ -0,0 +1,58 @@
+# Geotrellis Markdown Index
+
+Getting started with any library is a frustrating — often intimidating —
+experience. GeoTrellis is not an exception: perhaps you're not a Scala
+developer (or a developer at all) or maybe you know your way around code
+but are new to the specific domain of GIS software. Whatever the case,
+the collected notes, use-cases, and examples in this folder are intended
+to help you grapple with the various components which make up
+GeoTrellis.
+
+![Raster vs Vector](./img/596px-Raster_vector_tikz.png)
+
+- [geotrellis.etl](etl/etl-intro.md)
+- [geotrellis.gdal](gdal/gdal-intro.md)
+- [geotrellis.proj4](proj4/proj4-intro.md)
+- [geotrellis.raster](raster/raster-intro.md)
+  - [cell types](raster/celltype.md)
+  - [cloud removal](raster/cloud-removal.md)
+  - [interpolation](raster/interpolation.md)
+  - [map algebra](raster/map-algebra.md)
+  - [raster input/output](raster/raster-io.md)
+  - [rendering](raster/rendering.md)
+  - [resampling](raster/resampling.md)
+- [geotrellis.slick](spark/slick-intro.md)
+- [geotrellis.spark](spark/spark-intro.md)
+  - [indexing with the hilbert curve](spark/hilbert-curve.md)
```

Comment:
There's some links that are broken, that should point to existing files (like this should read `hilbert-index.md` instead of `hilbert-curve.md`).

I'm wondering if it's worth marking the ones that aren't currently complete.